framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,28 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
+     * This can be either a path alias (e.g. "@app/migrations/template.php")
+     * or a file path.
+     */
+    public $templateFileGenerators;
+    /**
+     * @var array the fields
+     */
+    public $fields;
+    /**
+     * @var string the primary key name
+     */
+    public $primaryKey = 'id';
+    /**
+     * @var string
+     */
+    public $createAt = 'create_at';
```

Comment:
Sorry for my english.

i asked if i should drop primaryKey var too?

My idea is the following : 

```
yii migrate/create CreatePost --fields=title:string
```

Generate: 

```php
$this->createTable('post', [
  'id' => $this->primaryKey(),
  'title' => $this->string()
]);
```

```
yii migrate/create CreatePost --fields=title:primaryKey
```

Generate: 

```php
$this->createTable('post', [
  'title' => $this->primaryKey(),
]);
```

is it correct?